data/state/account_test.go

```diff
@@ -242,3 +243,104 @@ func TestAccount_SetCodeHashWithJournal(t *testing.T) {
 	assert.Equal(t, 1, journalizeCalled)
 	assert.Equal(t, 1, saveAccountCalled)
 }
+
+func TestAccount_SetOwnerAddressWithJournal(t *testing.T) {
+	t.Parallel()
+
+	count := 0
+	tracker := &mock.AccountTrackerStub{
+		JournalizeCalled: func(entry state.JournalEntry) {
+			count++
+		},
+		SaveAccountCalled: func(accountHandler state.AccountHandler) error {
+			count++
+			return nil
+		},
+	}
+
+	acc, _ := state.NewAccount(&mock.AddressMock{}, tracker)
+
+	owner := []byte("owner")
+	err := acc.SetOwnerAddressWithJournal(owner)
+	require.Nil(t, err)
+	require.Equal(t, 2, count)
+	require.Equal(t, acc.OwnerAddress, owner)
+}
+
+func TestAccount_AddAndClaimDeveloperRewards(t *testing.T) {
+	t.Parallel()
+
+	tracker := &mock.AccountTrackerStub{
+		JournalizeCalled: func(entry state.JournalEntry) {
+		},
+		SaveAccountCalled: func(accountHandler state.AccountHandler) error {
+			return nil
+		},
+	}
+
+	acc, _ := state.NewAccount(&mock.AddressMock{}, tracker)
+	owner := []byte("owner")
+	_ = acc.SetOwnerAddressWithJournal(owner)
+
+	devRewards := big.NewInt(100)
+	err := acc.AddToDeveloperReward(devRewards)
+	require.Nil(t, err)
+
+	rewards, err := acc.ClaimDeveloperRewards([]byte("wrongAddr"))
```

Comment:
could split this in 2 different unit tests. one for claiming for a wrong address and one for happy scenario where it should work. Imagine that you make some changes and the happy path won't work anymore. If the test fails when a wrong address receives rewards, you cannot find out that the correct address received because you used `require`

Reply:
done.